tests/test_lib_policy.py

```diff
@@ -497,7 +497,7 @@ def test_17_ui_get_rights(self):
         P = PolicyClass()
         rights = P.ui_get_rights(SCOPE.USER, "realm2", "user")
         # there was still another policy...
-        self.assertEqual(rights, ["enable", "disable"])
+        self.assertEqual(set(rights), {"enable", "disable"})
```

Comment:
is there a reason why this can not be compares as a list anymore?
WOuld this change the program behaviour?

Reply:
Since ordering is part of the list, we can not (safely) compare lists
generated from dictionaries (since their order can not be guaranteed).
We should check If we compare lists like that somewhere in the code.
Unfortunately lists ans sets arent't the same since lists can contain
multiple occurrences of the same value.

Reply:
Correct.
BTW: `{"disable", "enable"}` denotes a `set`-literal, not a dictionary.